### PR TITLE
Version 1.9.2

### PR DESCRIPTION
**Pull request by AsparagusEduardo**
## Description
- Added missing 1.9.1 changelog from docs website.
- Fixed 1.9.0 changelog "full changes" calling 1.8.5 instead of 1.8.6.
- Added missing section for documentation in changelog template.
- Added Hardlocks and Compilation issues to "Softlock/Crash fixes" section.

## **Discord contact info**
AsparagusEduardo

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -178,7 +178,7 @@ With this, you'll get the latest version of pokeemerald-expansion, plus a couple
 - Check your current version.
     - You can check in the debug menu's `Utilities -> Expansion Version` option.
     - If the option is not available, you possibly have version 1.6.2 or older. In that case, please check the [changelogs](CHANGELOG.md) to determine your version based on the features available on your repository.
-- Once you have your remote set up, run the command `git pull RHH expansion/X.Y.Z`, replacing X, Y and Z with the digits of the respective version you want to update to (eg, to update to 1.8.4, use `git pull RHH expansion/1.8.4`).
+- Once you have your remote set up, run the command `git pull RHH expansion/X.Y.Z`, replacing X, Y and Z with the digits of the respective version you want to update to (eg, to update to 1.3.1, use `git pull RHH expansion/1.3.1`).
```
Comment (Pawkkie) [inline README.md:181]: Is there any reason this was changed to 1.3.1 as opposed to a current version?

File: docs/changelogs/1.9.x/1.9.2.md [hunk 2]
```diff
@@ -0,0 +1,178 @@
...
+
+## 🌋 *REFACTORS* 🌋
+📜 = Uses a migration script.
+* Remove unused `BattleScript_WindPowerActivatesEnd2` in [#5257](https://github.com/rh-hideout/pokeemerald-expansion/pull/5257)
+* Refactored in-battle disobedience to fix bug in [#5245](https://github.com/rh-hideout/pokeemerald-expansion/pull/5245)
+
+## 💥 *Hardlock/Softlock/Crash/Compiling fixes* 💥
+* Fixed hardlock when Hyperspace Fury is used by Hoopa Unbound by @AlexOn1ine in [#5237](https://github.com/rh-hideout/pokeemerald-expansion/pull/5237)
+* Fixed compile error when `OW_POKEMON_OBJECT_EVENTS` is `TRUE` but `P_HISUIAN_FORMS` is `FALSE` around Basculin by @hjk321 in [#5256](https://github.com/rh-hideout/pokeemerald-expansion/pull/5256)
+* Fixed hardlock when the AI cannot choose moves due to its opponing having Wonder Guard by @Pawkkie and Wiz in [#5317](https://github.com/rh-hideout/pokeemerald-expansion/pull/5317)
```
Comment (Pawkkie) [inline docs/changelogs/1.9.x/1.9.2.md:17]:
```suggestion
* Fixed hardlock when the AI cannot choose moves due to its opponent having Wonder Guard by @Pawkkie and Wiz in [#5317](https://github.com/rh-hideout/pokeemerald-expansion/pull/5317)
```

File: docs/changelogs/1.9.x/1.9.2.md [hunk 3]
```diff
@@ -0,0 +1,178 @@
...
+* Added Paldean Wooper and Clodsire overworld sprites by @Cafeei in [#5277](https://github.com/rh-hideout/pokeemerald-expansion/pull/5277)
+* Added missing Sirfetch'd competitive alias (`SPECIES_SIRFETCH_D`) by @cawtds in [#5283](https://github.com/rh-hideout/pokeemerald-expansion/pull/5283)
+* Added missing Gen 9 Overworld sprites by @Liamjd14 in [#5304](https://github.com/rh-hideout/pokeemerald-expansion/pull/5304)
+    * Original sprites by Darkus_Shadow, Princess-Phoenix, shaderr31, Molfang62, CarmaNekko, EduarPokeN, Larryturbo, TyranitarDark and Anarlaurendil
+    * Sources:
+        * Normal: https://www.deviantart.com/darkusshadow/art/Gen-9-Paldea-Pokemon-Overworld-Sprites-967776690
+        * Shiny: https://www.deviantart.com/darkusshadow/art/SHINY-Gen-9-Paldea-Pokemon-Overworld-Sprites-967779547
+### Changed
+* Improved Garganacl and Naclstack battle sprites by using the ones from @CyanSMP64's repo by @kittenchilly in [#5142](https://github.com/rh-hideout/pokeemerald-expansion/pull/5142)
+* Improved shiny both Indeedee by @Cafeei in [#5285](https://github.com/rh-hideout/pokeemerald-expansion/pull/5285)
```
Comment (Pawkkie) [inline docs/changelogs/1.9.x/1.9.2.md:39]:
```suggestion
* Improved both shiny Indeedee by @Cafeei in [#5285](https://github.com/rh-hideout/pokeemerald-expansion/pull/5285)
```

File: docs/changelogs/1.9.x/1.9.2.md [hunk 4]
```diff
@@ -0,0 +1,178 @@
...
+## 🧶 Items 🧶
+### Fixed
+* Fixes berries missing their timing after passive damage by @AlexOn1ine in https://github.com/rh-hideout/pokeemerald-expansion/pull/5300
+
+## 🤖 Battle AI 🤖
+### Changed
+* AI is encouraged to use "always crit" moves on partner with Anger Point by @SarnPoke in [#5244](https://github.com/rh-hideout/pokeemerald-expansion/pull/5244)
+### Fixed
+* Fixed AI not seeing the power of Max Moves by @AlexOn1ine in [#5299](https://github.com/rh-hideout/pokeemerald-expansion/pull/5299)
+* Fixed minor wrong order in `AI_CalcDamage` that made Nature Power not being considered for Z-Moves by @AlexOn1ine in [#5155](https://github.com/rh-hideout/pokeemerald-expansion/pull/5155)
```
Comment (Pawkkie) [inline docs/changelogs/1.9.x/1.9.2.md:152]:
```suggestion
* Fixed minor wrong order in `AI_CalcDamage` that made Nature Power not be considered for Z-Moves by @AlexOn1ine in [#5155](https://github.com/rh-hideout/pokeemerald-expansion/pull/5155)
```

File: README.md (hunk 1 above)
Comment (AsparagusEduardo) [reply README.md:181]: > Is there any reason this was changed to 1.3.1 as opposed to a current version?

A silly change, I'll change it to the current version :)

Comment (AsparagusEduardo) [reply README.md:181]:
```suggestion
- Once you have your remote set up, run the command `git pull RHH expansion/X.Y.Z`, replacing X, Y and Z with the digits of the respective version you want to update to (eg, to update to 1.9.2, use `git pull RHH expansion/1.9.2`).
```

**Comment by Pawkkie** [comment]
That's all I caught, looks good to me otherwise :)

**Comment by AsparagusEduardo** [comment]
Ready for review!

**Pull request merged**